app/app.scss

```diff
@@ -0,0 +1,163 @@
+@font-face {
```

Comment:
File is scss but it does not utilizes any feature of former. Let's use just simple css with `https://github.com/postcss/postcss` as additional loader in webpack. This way we will get rid of the ruby in project (sass is written in ruby) and get ability to use newest css features. What do you think?